aether-kernel/aether/kernel/api/models.py

```diff
@@ -677,7 +672,8 @@ def clean(self):
         else:
             self.modified = '{}-{}'.format(datetime.now().isoformat(), self.id)
 
-        if self.schema:
+        if self.schemadecorator:
+            self.schema = self.schemadecorator.schema  # redundant values taken from schema decorator
```

Comment:
What happens when there is no `schemadecorator`? How do we validate the `payload`? :thinking:

Reply:
This is what I mean

```python
        if self.schemadecorator:  # redundant values taken from schema decorator	
            self.schema = self.schemadecorator.schema
        if self.schema:
            # validate payload ....
```